app/models/shipit/task.rb

```diff
@@ -3,7 +3,11 @@ module Shipit
   class Task < Record
     include DeferredTouch
 
-    ConcurrentTaskRunning = Class.new(StandardError)
+    class ConcurrentTaskRunning < StandardError
```

Comment:
I saw that there are a bunch of other standard error subclasses that do not define `message`; e.g. `Error`, etc. Would those also need to be implemented for the `conflict` method in the other file... I am unsure if those other errors may get propogated all the way to that method.